### PR TITLE
Add support of Polish language

### PR DESCRIPTION
**Pull request by nickspring**
I've embedded http://github.com/blevesearch/stempel to bleavesearch because I needed changes here too (to embed *.tbl file in build I need load tier from bytes array, not only by filename). Unfortunately it's complicated to track 2 pull requests as I need it in my project right now.

### CODE REVIEW

**Comment by abhinavdangeti** [comment]
Thanks you for this contribution @nickspring.
The team will review this soon.

File: analysis/lang/pl/stemmer_pl.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+const SnowballStemmerName = "stemmer_pl"
+
+type PolishStemmerFilter struct {
+	trie stempel.Trie
+}
+
+func NewPolishStemmerFilter() *PolishStemmerFilter {
+	trie, err := stempel.LoadTrie()
+	if err != nil {
+		log.Fatal(err)
```
Comment (abhinavdangeti) [inline analysis/lang/pl/stemmer_pl.go:33]: Let's return an error here as opposed to log.Fatal.

File: analysis/lang/pl/stempel/fuzz.go [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+//go:build gofuzz
+// +build gofuzz
+
+package stempel
+
+var fuzzTrie Trie
+
+func init() {
+	var err error
+	fuzzTrie, err = Open("pl/stemmer_20000.tbl")
```
Comment (abhinavdangeti) [inline analysis/lang/pl/stempel/fuzz.go:24]: Would you throw in a comment here on what the contents of the file are?

**Review by abhinavdangeti — CHANGES_REQUESTED** [review]

File: analysis/lang/pl/stempel/fuzz.go (hunk 2 above)
Comment (nickspring) [reply analysis/lang/pl/stempel/fuzz.go:24]: It's prefix tree (trie) data, please see https://lucene.apache.org/core/6_6_0//analyzers-stempel/index.html?org/egothor/stemmer/class-use/Trie.html and https://github.com/blevesearch/bleve/issues/275 (@mschoch contribution)

File: analysis/lang/pl/stemmer_pl.go (hunk 1 above)
Comment (nickspring) [reply analysis/lang/pl/stemmer_pl.go]: done, thank you for review

**Review by abhinavdangeti — APPROVED** [review]

**Comment by abhinavdangeti** [comment]
@nickspring One last request (probably) here - would you rebase your change over master to resolve the unresolved conflicts. We can target this for v2.3.10 then.

**Comment by nickspring** [comment]
done

**Pull request merged**